src/views/RecIM/views/Activity/index.js

```diff
@@ -44,7 +46,14 @@ const Activity = () => {
       setLoading(false);
     };
     loadData();
-  }, [profile, activityID, openTeamForm, openCreateSeriesForm, openCreateMatchForm]);
+  }, [
+    profile,
+    activityID,
+    openTeamForm,
+    openCreateSeriesForm,
+    openCreateMatchForm,
+    openActivityForm,
+  ]);
```

Comment:
This is an example of how we force a page reload upon submit (specifically when the openActivityForm is changed from true->false)